### DISCUSSION TITLE
This is true music!

### DISCUSSION THREAD

**Discussion by wwwwsn147** [share121/inter-knot#1459] [其他]
![image](https://github.com/user-attachments/assets/3d2804da-9588-4e05-9115-db59a8a37b2a)

**Comment by wwwwsn147** [comment]
和我来一起享受音乐吧
https://www.bilibili.com/video/BV1hq4y1s7VH?vd_source=fb8283747c211b184054c34c65e07760

**Reply by Capnse5** [reply]
I was scammed😢

**Comment by zxs285f** [comment]
太好听辣

**Comment by fadai0528** [comment]
666

**Comment by natihawi** [comment]
提醒大家！这是真正的*%#好音乐！

**Comment by MissHedwig** [comment]
大吉大利！

**Comment by Kamisato100** [comment]
早知道，还是原道<img src="https://img-static.mihoyo.com/communityweb/upload/eae93728d054ff260942518c488a39e2.png" alt="大哭" width="75" >

**Comment by Jshuya24** [comment]
这是谁家小孩

**Comment by ruanmiaomiao** [comment]
我  就  知  道

**Comment by Sheng-gy** [comment]
又是被骗的一天（ ᗜ ‸ ᗜ ）

**Comment by MrDucker** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >可恶被骗了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/189d73f83e4892766ac8197f2df156d0_3183903856565806928.png" alt="比利-呜呜呜" width="75" >

**Comment by Genshin-Neuvillette** [comment]
谁家小孩？

**Comment by anbi-demala** [comment]
我好像被骗了，又好像没有<img src="https://upload-bbs.miyoushe.com/upload/2023/01/17/9c6372ecac2a16b11f3680194d00870d_7405142885051349596.png" alt="疑惑呜呜" width="75" >

**Comment by 00zzxx** [comment]
如果我知道我会被骗那我算不算被骗了呢<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/06f3f4c54fc51d82ba3e575194504d80_6167770388617778929.png" alt="米游君-喝可乐" width="75" >

**Comment by Yoimiya-M** [comment]
**豪庭**
